examples/all-clusters-app/all-clusters-common/all-clusters-app.zap

```diff
@@ -4994,6 +4994,97 @@
             }
           ]
         },
+        {
+          "name": "Air Pressure Measurement",
```

Comment:
This should be on endpoint 1 instead of (or at least in addition to) endpoint 0, in order for CI to pass.